File: euler003.js
let factors =[];

function isPrimeFactor(n) {
	for (var i = 2; i <= n; i++) {
		if (n%i===0) {
			n=n/i;
			factors.push(i);

		} else {
			i = i +1;

		}
		
	}
}
isPrimeFactor(600851475143);
console.log(Math.max(factors));

File: euler003.py
factors = []

def isPrimeFactor(n):
	i=2
	while i <= n:
		if n%i ==0:
			n = n/i
			factors.append(i)
		else:
			i = i + 1
isPrimeFactor(600851475143)
print factors
print (int(max(factors)))